### PR TITLE
6383: fixed ordering of data from the CapabilityRepository. Added fix…

### PR DESCRIPTION
**Pull request by SimonLLawrence**
… for failing spectflow

### CODE REVIEW

File: src/NHSD.BuyingCatalogue.API/NHSD.BuyingCatalogue.API.csproj [hunk 1]
```diff
@@ -29,6 +29,7 @@
     <PackageReference Include="Microsoft.Extensions.Diagnostics.HealthChecks.Abstractions" Version="3.1.1" />
     <PackageReference Include="Serilog.AspNetCore" Version="3.2.0" />
     <PackageReference Include="Serilog.Sinks.Console" Version="3.1.1" />
+    <PackageReference Include="SpecFlow" Version="3.1.78" />
```
Comment (sam-roberts-bjss) [inline src/NHSD.BuyingCatalogue.API/NHSD.BuyingCatalogue.API.csproj:32]: Please review this additional package as I am not sure it is needed in the BuyingCatalogue.Api prorject.

Comment (SimonLLawrence) [reply src/NHSD.BuyingCatalogue.API/NHSD.BuyingCatalogue.API.csproj]: Removed import and fixed failing test

**Comment by SimonLLawrence** [comment]
AB#6325

**Comment by SimonLLawrence** [comment]
AB#6383

**Pull request merged**